### PR TITLE
[New Rule] Potential Process Masquerading as WerFault

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/209

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/defense_evasion_masquerading_werfault.toml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+risk_score = 47
+rule_id = "6ea41894-66c3-4df7-ad6b-2c5074eb3df8"
+severity = "medium"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category :"process" and not event.type:"end" and
+ process.name:"WerFault.exe"  and
+ not process.args:((("-u" or "-pss") and "-p" and "-s") or ("/h" and "/shared") or ("-k" and "-lcq"))
```
Comment (brokensound77) [inline rules/windows/defense_evasion_masquerading_werfault.toml:31]:
```suggestion
event.category:process and not event.type:end and
 process.name:WerFault.exe  and
 not process.args:((("-u" or "-pss") and "-p" and "-s") or ("/h" and "/shared") or ("-k" and "-lcq"))
```

File: rules/windows/defense_evasion_masquerading_werfault.toml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2020/08/24"
+ecs_version = ["1.5.0"]
```
[the remark this answers is not in the document]
Comment (Samirbous) [reply rules/windows/defense_evasion_masquerading_werfault.toml:3]: @brokensound77  FYI all other PRs failed for same test (apparently ECS schema version 1.6 not available in the unit testing script)

 image

**Review by devonakerr — APPROVED** [review]

File: rules/windows/defense_evasion_masquerading_werfault.toml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    "https://www.hexacorn.com/blog/2019/09/20/werfault-command-line-switches-v0-1/",
+]
+risk_score = 47
+rule_id = "6ea41894-66c3-4df7-ad6b-2c5074eb3df8"
+severity = "medium"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:process and not event.type:end and
```
Comment (brokensound77) [inline rules/windows/defense_evasion_masquerading_werfault.toml:29]:
```suggestion
event.category:process and event.type:(start or process_started) and
```

File: rules/windows/defense_evasion_masquerading_werfault.toml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+ process.name:WerFault.exe  and
```
Comment (brokensound77) [inline rules/windows/defense_evasion_masquerading_werfault.toml:30]:
```suggestion
 process.name:WerFault.exe and
```

**Review by brokensound77 — APPROVED** [review]
2 suggestions then LGTM

File: rules/windows/defense_evasion_masquerading_werfault.toml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+creation_date = "2020/08/24"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/08/24"
+
+[rule]
+author = ["Elastic"]
+description = """
+A suspicious WerFault instance command line was detected, which may indicate an attempt to run unnoticed. Verify
+process details such as command line, network connections, file writes and parent process details as well.
```
Comment (bm11100) [inline rules/windows/defense_evasion_masquerading_werfault.toml:11]:
```suggestion
Identifies a suspicious WerFault command line parameter, which may indicate an attempt to run unnoticed. 
```
Would move to `note`- 
> Verify process details such as command line, network connections, file writes, and parent process information.

**Pull request merged**